src/main/java/org/apache/maven/resolver/internal/ant/Names.java

```diff
@@ -19,22 +19,38 @@
 package org.apache.maven.resolver.internal.ant;
 
 /**
+ * Centralized constants used throughout the Maven Resolver Ant tasks.
+ * <p>
+ * This class defines symbolic names for commonly used IDs, property keys,
+ * and filenames used in the integration with Ant, such as default repository identifiers,
+ * offline mode flags, and POM registration keys.
+ * </p>
+ *
+ * <p>
+ * This class is not intended to be instantiated.
+ * </p>
  */
 public final class Names {
 
     private Names() {
         // hide constructor
     }
 
+    /** ID namespace prefix used by the resolver: {@code "resolver"}. */
     public static final String ID = "resolver";
 
+    /** Default ID for the shared repository list: {@code "resolver.repositories"}. */
```

Comment:
Maybe it's an ant thing, but I don't understand "ID" in this class. What are these IDs and where are they used?

Reply:
These are constants used when the type is a reference. Used in E.g. AntRepoSys.getInstance(..), AuetherUtils.getDefaultRepositories(..), AntRepoSys.initDefaults() etc. An object in ant can exist as the Object instance itself and as a "reference" to an instance that can be resolved with project.getReference(). In the cases where these constants are used, we use a static reference to ensure a "singleton" like behavior in the project.